docs/source/clickhouse.md

```diff
@@ -0,0 +1,98 @@
+# ClickHouse
+
+[ClickHouse](https://clickhouse.com/docs/en/intro) is a fast and efficient column-oriented database for analytical workloads, making it easy to analyze Hub-hosted datasets with SQL. To get started quickly, use [`clickhouse-local`](https://clickhouse.com/docs/en/operations/utilities/clickhouse-local) to run SQL queries from the command line and avoid the need to fully install ClickHouse.
+
+<Tip>
+
+Learn more about the Hugging Face and ClickHouse integration in this blog post for more details about how to analyze datasets on the Hub with ClickHouse.
+
+</Tip>
+
+To start, download and install `clickhouse-local`:
+
+```bash
+curl https://clickhouse.com/ | sh
+```
+
+For this example, you'll analyze the [maharshipandya/spotify-tracks-dataset](https://huggingface.co/datasets/maharshipandya/spotify-tracks-dataset) which contains information about Spotify tracks. Datasets on the Hub are stored as Parquet files and you can access it with the [`/parquet`](parquet) endpoint:
+
+```py
+import requests
+
+r = requests.get("https://datasets-server.huggingface.co/parquet?dataset=maharshipandya/spotify-tracks-dataset")
+j = r.json()
+url = [f['url'] for f in j['parquet_files']]
+url
+['https://huggingface.co/datasets/maharshipandya/spotify-tracks-dataset/resolve/refs%2Fconvert%2Fparquet/maharshipandya--spotify-tracks-dataset/csv-train.parquet']
+```
+
+## Aggregate functions
+
+Now you can begin to analyze the dataset. Use the `-q` argument to specify the query to execute, and the [`url`](https://clickhouse.com/docs/en/sql-reference/table-functions/url) function to create a table from the data in the Parquet file.
+
+Let's start by identifying the most popular artists:
+
+```bash
+./clickhouse local -q "
+    SELECT count() AS c, artists 
```

Comment:
this endpoint is currently broken, even when the data is present we will need

SETTINGS enable_url_encoding = 0, max_http_get_redirects = 1